Framework/Geometry/inc/MantidGeometry/Objects/ShapeFactory.h

```diff
@@ -83,8 +107,8 @@ class MANTID_GEOMETRY_DLL ShapeFactory {
   std::string parseInfiniteCylinder(Poco::XML::Element *pElem, std::map<int, std::shared_ptr<Surface>> &prim,
                                     int &l_id);
   std::string parseCylinder(Poco::XML::Element *pElem, std::map<int, std::shared_ptr<Surface>> &prim, int &l_id);
-  std::string parseSegmentedCylinder(Poco::XML::Element *pElem, std::map<int, std::shared_ptr<Surface>> &prim,
-                                     int &l_id);
+  // std::string parseSegmentedCylinder(Poco::XML::Element *pElem, std::map<int, std::shared_ptr<Surface>> &prim,
+  //                                    int &l_id);
```

Comment:
Please can you either delete these lines or if they're were temporarily commented out reinstate?

Reply:
This method has no references and the SegmentedCylinder entry in ShapeInfo::GeometryShape was removed after this convo: https://github.com/mantidproject/mantid/pull/21310#discussion_r154614470

So I'll delete the parseSegmentedCylinder() method